### DISCUSSION TITLE
Auto Run on Windows boot not working on windows 11

### DISCUSSION THREAD

**Discussion by muneebshafiq** [zhongyang219/TrafficMonitor#885] [Q&A]
Any idea how to make auto start work on Windows 11 as I have enabled option but program doesn't start automatically. Also app doesn't appear on Task Manager > Startup

**Comment by medontcode** [comment]
I have it running on 2 Windows 11 PCs and TMon starts automatically everytime I turn them on.
I just checked and TMon is listed in "Task Manager > Startup" for both my PCs.

<img src="https://i.imgur.com/YNckdUd.jpg" >

**Reply by muneebshafiq** [reply]
Strange! It doesn't work for me on Surface Laptop 3!
Do I have to run program with admin privileges?

**Reply by muneebshafiq** [reply]
Anyhow I manually added the shortcut to startup folder!

**Reply by soloopooo** [reply]
> Strange! It doesn't work for me on Surface Laptop 3! Do I have to run program with admin privileges?

Mine is with temperature so i must use it with admin privileges and that app autostart works for me.

**Reply by Ahasan100** [reply]
please tell him #zhongyang219 write about section. it shows an old version 1.84.1, but I alteady installed 1.85.

**Comment by medontcode** [comment]
Try this and restart your device once you have made the changes...

<img src="https://i.imgur.com/fIVfFo4.jpg" >
